pages/results/index.tsx

```diff
@@ -24,6 +24,8 @@ const ResultsPage = dynamic(
   { ssr: false }
 )
 
+// just for testing docker on teamcity
```

Comment:
we can remove this comment now.

Reply:
wasn't not planning on merging this change